docs/accessing_transaction_field.rst

```diff
@@ -7,43 +7,60 @@ PyTeal smart contracts can access properties of the current transaction and the
 blockchain when they are running.
 
 Transaction Fields
-------------------
+##################
```

Comment:
I think something's off about the title changes.

Before this PR:
![Screen Shot 2022-02-18 at 12 23 14 PM](https://user-images.githubusercontent.com/5856867/154756059-a9e2cc58-f01e-4c1a-8e6c-b4406f421f7a.png)

After this PR:
![Screen Shot 2022-02-18 at 12 23 22 PM](https://user-images.githubusercontent.com/5856867/154756071-9b4f40d8-7795-43e9-9960-d4bd0a6c6a25.png)

If you want pointers on debugging this, let me know

Reply:
@jasonpaulos Appreciate the offline pointers on how to debug HTML generation.  I set a reminder to open a small, separate PR documenting the steps.

Via https://github.com/algorand/pyteal/pull/204/commits/4631bba0e3cf8d5d3440e77d97dc30f96fe2256c, I believe I fixed the header level indentation.  I felt adding a _Fields by Transaction Type_ header helped.  If you feel otherwise, I'll remove it.

Additionally, I opted to alphabetize the table listing excluding _Common Fields_:  https://github.com/algorand/pyteal/pull/204/commits/673675bfe4abf1931622e31dff5f208f4e32af89.